### ISSUE TITLE
`nil` is not redundant when initializing lazy properties

### ISSUE THREAD

**Issue by tadeuzagallo** [nicklockwood/SwiftFormat#135]
I understand that the following snippet makes little to no sense, but it's valid and I only found it because SwiftFormat removed the `nil` and then it was no longer valid. It's a bit of an edge case, but I thought I'd report it anyway.

```swift
// foo.swift
import Foundation
class A {
  lazy var x : A? = nil
}
```

Fine:
```bash
swift < foo.swift
```

Not fine:
```bash
swiftformat < foo.swift | swift
```

**Comment by nicklockwood** [comment]
Thanks, should be a simple fix.

**Comment by nicklockwood** [comment]
Fixed in 0.28.5
